pkg/Library/App/Resources.js

```diff
@@ -33,6 +33,36 @@ export const Resources = {
   }
 };
 
+const resources2 = {};
```

Comment:
what is this used for vs regular `resources` above?

Reply:
Experimental: my instincts tell me the 'version' concept should be able to replace the subscription model for streams (better because it requires no _unsubscribing_), but my brain is struggling with the realities. This is for experimenting with it.